File: python/leetcode/find-peak-element.py
class Solution:
    def findPeakElement(self, nums):
        for i in range(len(nums)):
            if (i - 1 < 0 or nums[i - 1] < nums[i]) and (i + 1 >= len(nums) or nums[i] > nums[i + 1]):
                return i
        return False

File: java/leetcode/find-peak-element.java
class Solution {
    public int findPeakElement(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            if ((i - 1 < 0 || nums[i - 1] < nums[i]) && (i + 1 >= nums.length || nums[i] > nums[i + 1])) {
                return i;
            }
        }
        return 0;
    }
}
